seqr/views/utils/export_utils.py

```diff
@@ -96,9 +96,9 @@ def export_multiple_files(files, zip_filename, file_format='csv', add_header_pre
             for filename, header, rows in files:
                 header_display = header
                 if add_header_prefix:
-                    header_display = map(
-                        lambda (i, k): '{}-{}'.format(str(i).zfill(2), k), enumerate(header)
-                    )
+                    header_display = list(map(
+                        lambda header_tuple: '{}-{}'.format(str(header_tuple[0]).zfill(2), header_tuple[1]), enumerate(header)
+                    ))
```

Comment:
I think `list(map(...))` is less readable than a list comprehension (and honestly I'm not sure why we were using `map` in the first place) so I would change this to 
```
header_display = [{}-{}'.format(str(header_tuple[0]).zfill(2), header_tuple[1]) for header_tuple in enumerate(header)]
```

Reply:
Okay